HISTORY.md

```diff
@@ -1,3 +1,16 @@
+# Release 0.24.0
```

Comment:
Should we put in the release notes the minimal supported wheels for macOS is now Big Sur (ver. 11)?

Reply:
Will do, thanks